src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -500,19 +500,19 @@
                     }
                     else
                     {
-                        var diffRevisionForButton = Model.ReviewContent.APIRevisions[diffRevisionTypeFirst];
+                        var diffRevisionForButton = Model.ReviewContent.APIRevisionsGrouped[diffRevisionTypeFirst].OrderByDescending(c => c.CreatedOn);

```

Comment:
is there a way that you can sort the data out of the DB so we don't need to sort it here all the time? 
or just sort it once?

Reply:
Fixed